src/js/transfers/bridged-nep21-to-erc20/index.js

```diff
@@ -17,53 +23,97 @@ export async function initiate (nep21Address, amount) {
     { changeMethods: ['withdraw'] }
   )
 
-  // TODO: can cause redirect to NEAR Wallet, rendering `tx` inaccessible!
-  const tx = await bridgeToken.withdraw(
+  const withdrawTxHash = initializeTransaction(bridgeToken.withdraw(
```

Comment:
I think this approach more or less can be implemented as is with current near-api-js:
```js
const [withdrawTxHash, withdrawTx] = account.signTransaction(bridgeTokenId, [functionCall('withdraw', {
       amount: String(amount),
       recipient: window.ethUserAddress.replace('0x', '')
     },
     new BN('3' + '0'.repeat(14)) // 10x current default from near-api-js
)]);
```

then persist either hash or transaction hash if needed and send using provider.

I guess main problem is that a lot of retry logic, etc is inside of `account.signAndSendTransaction`. Probably makes sense to move everything downstream (except nonce retries).